content/post/2022-07-28-gsoc-ldr426-intro/index.md

```diff
@@ -0,0 +1,62 @@
+---
+# Documentation: https://sourcethemes.com/academic/docs/managing-content/
+title: "Hello from Dingrui Lei, GSoC contributor on Echoshader!"
+date: 2022-07-28T00:00:00-08:00
+authors: 
+  - Dingrui Lei
+  - landungs
```

Comment:
Are these suppose to be github handles? if so please change to `ldr426` and `lsetiawan`

Reply:
No, this renders to be author list. lsetiwan links to your profile automatically at the end of post, Dingrui doesn't have a profile so it's just his name.

Reply:
Ah I see. Cool. Thanks for letting me know 😄